### PR TITLE
Update Dependencies

### PR DESCRIPTION
**Pull request by sdkrach**
The build currently fails because of unresolvable dependencies. This PR updates the dependencies, particularly the TP to the recent version.

### CODE REVIEW

File: bundles/tools.mdsd.modelingfoundations.identifier/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -7,11 +7,12 @@ Bundle-Version: 0.1.0.qualifier
...
-Bundle-RequiredExecutionEnvironment: JavaSE-10
+Bundle-RequiredExecutionEnvironment: JavaSE-11
 Export-Package: tools.mdsd.modelingfoundations.identifier,
  tools.mdsd.modelingfoundations.identifier.impl,
  tools.mdsd.modelingfoundations.identifier.util
 Require-Bundle: org.eclipse.emf.ecore;visibility:=reexport,
  org.eclipse.core.runtime,
- org.eclipse.emf.cdo
+ org.eclipse.emf.cdo,
+ tools.mdsd.modelingfoundations.workflow;bundle-version="0.1.0";resolution:=optional
```
Comment (seiferma) [inline bundles/tools.mdsd.modelingfoundations.identifier/META-INF/MANIFEST.MF:17]: Why is there a dependency from the model bundle to the workflow bundle? Even if the resolution is optional, I do not see why this is necessary.

**Review by seiferma — CHANGES_REQUESTED** [review]
Looks good to me but I do not understand the dependency between the model and the workflow bundle.

File: bundles/tools.mdsd.modelingfoundations.identifier/META-INF/MANIFEST.MF (hunk 1 above)
Comment (sdkrach) [reply bundles/tools.mdsd.modelingfoundations.identifier/META-INF/MANIFEST.MF:17]: The optional dependency between model and workflow is required for tycho to order the build of the workflow plugin ahead of the model plugin.
Alternatively, we could get rid of the workflow bundle entirely and merge it into the model bundle. But then I think, we need to include the MWE2 dependencies.

Comment (seiferma) [reply bundles/tools.mdsd.modelingfoundations.identifier/META-INF/MANIFEST.MF:17]: Actually I like the separation of workflow and meta model. Other options to enforce the order in the reactor are either more error prone or more complex. Therefore, this seems to be the best solution.

**Review by seiferma — APPROVED** [review]

**Pull request merged**